src/mongo-c-performance.h

```diff
@@ -23,9 +23,9 @@
 #include <stddef.h>
 #include <assert.h>
 
-const int NUM_ITERATIONS;
-const int NUM_DOCS;
-char *g_test_dir;
+extern const int NUM_ITERATIONS;
+extern const int NUM_DOCS;
+extern char *g_test_dir;
```

Comment:
Without `extern` compilation with clang 11 gave duplicate symbol errors, like the following:

```
FAILED: mongo-c-performance 
: && /Users/kevin.albertson/bin/llvm-11.0.0/bin/clang -g -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  CMakeFiles/mongo-c-performance.dir/src/bson-performance.c.o CMakeFiles/mongo-c-performance.dir/src/main.c.o CMakeFiles/mongo-c-performance.dir/src/mongo-c-performance.c.o CMakeFiles/mongo-c-performance.dir/src/driver-performance.c.o CMakeFiles/mongo-c-performance.dir/src/gridfs-performance.c.o CMakeFiles/mongo-c-performance.dir/src/gridfs-parallel-performance.c.o CMakeFiles/mongo-c-performance.dir/src/ldjson-performance.c.o -o mongo-c-performance  -Wl,-rpath,/Users/kevin.albertson/install/mongo-c-driver-master/lib  /Users/kevin.albertson/install/mongo-c-driver-master/lib/libmongoc-1.0.0.0.0.dylib  /Users/kevin.albertson/install/mongo-c-driver-master/lib/libbson-1.0.0.0.0.dylib && :
duplicate symbol '_g_test_dir' in:
    CMakeFiles/mongo-c-performance.dir/src/bson-performance.c.o
    CMakeFiles/mongo-c-performance.dir/src/main.c.o
```